PolishWithBlondes/src/App.scss

```diff
@@ -1,12 +1,18 @@
-$primary-color: #3498db;
+* {
+  border: 1px solid red;
+}
```

Comment:
It's good that you use it for development purposes but please don't merge it to the main branch. (Remove this before each PR)

Reply:
Will do. Resolved